include/Editor.h

```diff
@@ -51,6 +51,7 @@ class Editor : public QMainWindow
 public:
 	void setPauseIcon(bool displayPauseIcon=true);
 	QAction *playAction() const;
+	inline static Editor* s_lastPlayedEditor = nullptr;
```

Comment:
Public data members shouldn't use the `s_` prefix.

Maybe you could make this private and introduce a public getter like this?
```cpp
static Editor* lastPlayedEditor() { return s_lastPlayedEditor; }
```

It looks like nothing outside of `Editor` needs to modify it, so a getter should be fine.